src/main/java/group1/intern/controller/seller/ProductsController.java

```diff
@@ -49,6 +62,29 @@ public String showProductDetailSeller(@PathVariable("id") Integer id, Model mode
             model.addAttribute("productInfo", productInfo);
             model.addAttribute("productDetailColors", productDetailColors);
             return "screens/seller/products/show";
+        } catch (NotFoundObjectException ex) {
+            model.addAttribute("toastMessages", new ToastMessage("error", ex.getMessage()));
+            return "screens/seller/products/index";
+        }
+    }
+
+    @GetMapping("/{id}/edit")
+    public String showEditProductDetail(@PathVariable("id") Integer id, Model model) {
```

Comment:
```suggestion
    public String editProductDetail(@PathVariable("id") Integer id, Model model) {
```
ri dc rồi, còn show edit nữa chứ =))